### DISCUSSION TITLE
play audio instantly

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [savonet/liquidsoap#2447] [User Support]
I'm starting with LS and I have a little doubt about playing audio immediately above the current audio and then the current audio starts playing again.

For example, a song is playing and when I run the command to play an audio, it enters above the current audio it plays, after I stop the audio that is already playing it continues.
This audio I want to play right away, I'm not sure if it can play without being in a playlist or if it can play normally.

**Comment by codeofdusk** [comment]
Perhaps show a minimal script example?

**Reply by Marcioghiraldelli** [reply]
radio = fallback(id="safe_fallback", track_sensitive = false, [radio, single(id="hour", "/usr/local/share/icecast/web/hour.mp3")])

**Reply by Marcioghiraldelli** [reply]
so that this audio should only enter the programming when I run the script.
That is, I ran the script or telnet command, it already enters the audio after a few seconds.

**Comment by Marcioghiraldelli** [comment]
I don't have the script ready, I need to know where, in what part of the documentation or if you can help me, how to do this.
I wanted a command, maybe where I can stop the current audio and enter a chosen audio and after the chosen audio ends the current audio continues playing.

**Comment by toots** [comment]
The kind of audio you want to play needs to be explained but, in general, `fallback` with `track_sensitive` set to `false` is the way to go for that. If needed, you can also write your own condition callback for a `switch` operator, if your use-case is more complex.

But, yeah, some more details would help point out to a specific solution.

**Comment by Marcioghiraldelli** [comment]
hello @toots 
Thanks for the answer.

what i need to know is
I have a station currently playing a song and I urgently need an announcement to interrupt the current song and at the end of the announcement, the current song will resume playing where it left off. That's basically it.

I don't know if you know but in azuracast for example there is an option called play where you select an audio and this audio after a few seconds enters the programming interrupting the current audio and at the end the current audio returns where it left off.

**Comment by Marcioghiraldelli** [comment]
radio = fallback(id="safe_fallback", track_sensitive = false, [radio, single(id="hour", "/usr/local/share/icecast/web/hour.mp3")])
I think of something like this.

**Reply by toots** [reply]
Yeah that sounds like it. Have you seen our [liquidsoap book](https://www.liquidsoap.info/doc-2.0.4/book.html)? It should have a lot of background and example to help you build your script..
